tests/unit_tests/test_picked_protein.py

```diff
@@ -0,0 +1,17 @@
+"""Test the picked protein approach functions"""
+import pandas as pd
+
+from mokapot.picked_protein import strip_peptides
+
+
+def test_strip_peptides():
+    """Test removing modifications from a peptide sequence"""
+    in_df = pd.Series(["A.B.C", "nABCc", "BL[+mod]AH"])
```

Comment:
Question: Would this behavior be a problem for older search engines that have the flanking amino acids with periods (AAs not in the peptide sequence, but in the protein seq)?

Reply:
This is actually to help with those. This function removes the flanking amino acids, because we don't want to consider them when mapping them to proteins and such.